SU-SWS/stanford_media
# Hsd8 651 google drive video

# READY FOR REVIEW

# Summary
- Added video provider for google drive videos.

# Needed By (Date)
- no rush

# Urgency
- low

# Steps to Test
1. upload a video to google drive and make it public to the web. [this guide helps](https://support.google.com/blogger/thread/1950766?hl=en)
1. grab the url for the video from the guide above.
1. use suhumsci repo for this
1. checkout this branch
1. clear cache or sync any site
1. create a basic page with a text area
1. click the media button in the wysiwyg
1. click the "external embed" tab at the top
1. past the link from earlier
1. save the page
1. ensure the video is embeded.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
